ecosystem/python/sdk/README.md

```diff
@@ -1 +0,0 @@
-The contents of this directory have been moved to the following repository: https://github.com/aptos-labs/aptos-python-sdk
```

Comment:
I feel like this kind of thing is still handy in case we've got legacy docs / readmes.

Reply:
Yeah... Well, I'm just trying to get rid of as much as possible.  Let it burn!